src/main/java/com/kyzegs/stattransfer/StatTransfer.java

```diff
@@ -38,11 +41,21 @@ public void run() {
 				try {
 					StatTransferUtil.transfer();
 				} catch (JSchException | SftpException e) {
-					getLogger().warning("SFTP settings in config.yml are most likely incorrect");
+					getLogger().warning(getString("log.JSchSftpExceptionWarning"));
 					e.printStackTrace();
 					cancel();
 				}
 			}
 		}.runTaskTimer(this, 0, 6000);
 	}
+
+	public String getString(String key) {
+		try {
+			return ResourceBundle
+					.getBundle("com.kyzegs.stattransfer.locale." + getConfig().getString("language", "english"))
```

Comment:
We should lower case the language that is specified in the `config.yml` here. Just so user's can't fuck up by typing it in upper case